src/main/java/com/google/devtools/build/lib/rules/python/PyBuiltins.java

```diff
@@ -266,44 +266,6 @@ public Object expandLocationAndMakeVariables(
         .expand(attributeName, expression);
   }
 
-  // TODO(b/232136319): Remove this once the --experimental_build_transitive_python_runfiles
-  // flag is flipped and removed.
-  @StarlarkMethod(
-      name = "new_empty_runfiles_with_middleman",
```

Comment:
I'll loop in @rickeylev -- this looks reasonable since it's VERY niche, but I'm not up to date about the details of the Python rules, so I'd prefer him to make the call.